spec/xliffer/file_spec.rb

```diff
@@ -100,6 +100,15 @@ module XLIFFer
           expect(subject.target_language).to eq('en')
         end
       end
+
+      describe 'get_attribute' do
+        it 'allows getting arbitrary attributes' do
+          xml_text = '<xliff><file foo="bar"></file></xliff>'
+          file_node = Nokogiri::XML.parse(xml_text).xpath('//file').first
+          subject = XLIFF::File.new file_node
+          expect(subject.get_attribute('foo')).to eq('bar')
```

Comment:
Could you add an test demonstrating the namespace-stripping behaviour?

Reply:
Sure: https://github.com/WOVNio/xliffer/pull/5/commits/1cf678ba34ffb1259e676f6e6daf13b2f650a6b9

Reply:
Thank you!